wurst/_wurst/Orders.wurst

```diff
@@ -0,0 +1,379 @@
+package Orders
+import NoWurst
+
+/** Class that contains every known order in game that has no string counterpart */
+public class SpecialOrders
+	static constant tomeOfAttack			 = 852259
+	static constant wandOfIllusion			 = 852274
+
+
+
+/*
```

Comment:
please remove this and just add 1 line of credit to the guy after imports like other packages